### ISSUE TITLE
manufacture-SPC app fixes

### ISSUE THREAD

**Issue by ycaokris** [plotly/dash-sample-apps#212]
# App name
Manufacture-spc-dashboard

# Descriptions

Discussed with @shammamah  and @jackparmer , 
This app has a disabled tab.  It currently direct from setting view and proceed to measurement views sequentially, which makes tab misleading. For better user experience, changes below needs to be done:
* Change page-loading tab to measurement view and pre-load some data as initial point
* Restore tab-switching, Use dcc.Store to cache measurement states and parameter setting(machine needs to be stopped while on other tab)

* Add 'learn more' panel/button to describe app usage.

**Comment by ohaanika** [comment]
Awesome, I had the same thoughts regarding how tabs felt misleading! About changes, do you mean to only keep the "Proceed to Measurement" button and remove tabs entirely? Or have the ability to switch between tabs and essentially restart the measurement view if tab is switched back to setting view and values are changed? (Just checking!)

On a separate note, I noticed that [this portion of code (100+ lines)](https://github.com/plotly/dash-sample-apps/blob/fb1afe0b178c07cee8ec278851b78581020fb79a/apps/dash-manufacture-spc-dashboard/app.py#L1051-L1181) can be reduced to the following by simply implementing a for loop as follows:
```
for param in params[1:]:
    @app.callback(
        output=[
            Output(param + suffix_count, "children"),
            Output(param + suffix_sparkline_graph, "extendData"),
            Output(param + suffix_ooc_n, "children"),
            Output(param + suffix_ooc_g, "value"),
            Output(param + suffix_indicator, "color"),
        ],
        inputs=[Input("interval-component", "n_intervals")],
        state=[State("value-setter-store", "data")],
    )
    def update_param_row(interval, stored_data, param=param):
        count, ooc_n, ooc_g_value, indicator = update_count(interval, param, stored_data)
        spark_line_data = update_sparkline(interval, param)
        return count, spark_line_data, ooc_n, ooc_g_value, indicator
```

**Comment by ycaokris** [comment]
> Or have the ability to switch between tabs and essentially restart the measurement view if tab is switched back to setting view and values are changed?  

💯Thanks for checking it out. I've removed proceed button and enable tab-switch in https://github.com/plotly/dash-sample-apps/pull/129/commits/f6dca9ff171f7a00621f8951b400cce402cc3e46 and https://github.com/plotly/dash-sample-apps/pull/129/commits/9bf14fc7432dd44629c822ad42f2b0362f759419. Edits in tab1 has already been saved in one dcc.Store to update measurement scales. What has been changed in these commits was adding a dcc.Store to save stopped points whenever switch from measurement -> settings, and restore it when turn back. 

Great hack on loop, I'll definitely give a try and add it. More has been discussed in https://community.plot.ly/t/create-callback-using-for-loop/11147/2 and https://community.plot.ly/t/handel-multiple-callback-outputs-programmatically/4759/2. IMO don't have to add as example in user docs. 

There's still css needs to be fine-tuned. Let's sync up!
